Tests/Tests.iOS/Tests.iOS.csproj

```diff
@@ -28,7 +28,7 @@
     <CodesignKey>iPhone Developer</CodesignKey>
     <DeviceSpecificBuild>false</DeviceSpecificBuild>
     <MtouchNoSymbolStrip>true</MtouchNoSymbolStrip>
-    <MtouchExtraArgs>--linkskip=nunit.runner.iOS</MtouchExtraArgs>
+    <MtouchExtraArgs>--linkskip=nunit.runner.iOS --linkskip=Realm</MtouchExtraArgs>
```

Comment:
I'm trying to see if we can go around the whole inliner dumpster fire like this.